webapp/src/Entity/Problem.php

```diff
@@ -109,6 +109,21 @@ class Problem extends BaseApiEntity implements
     #[Serializer\Exclude]
     private ?string $problemstatement_type = null;
 
+    #[ORM\Column(options: [
+        'comment' => 'Whether this problem is a multi-pass problem.',
+        'default' => 0,
+    ])]
+    #[Serializer\Exclude]
+    private bool $isMultipassProblem = false;
+
+    #[ORM\Column(
+        nullable: true,
+        options: ['comment' => 'Optional limit on the number of rounds for multi-pass problems; defaults to 2 if not specified.', 'unsigned' => true]
+    )]
+    #[Assert\GreaterThan(0)]
+    #[Serializer\Exclude]
+    private ?int $multipassLimit = null;
```

Comment:
I think the code becomes much easier if we already set this to 2 (like the comment implies). Now this is only a constant which we use in other parts of the code.

It would lose the information if a problem uses the `default` value but I don't think we'll need that often in practice?

Reply:
Fair point, this was leaking info into other classes. I changed just the comment here and the getter below to do the right thing.